Astro 5/2 release notes (#2216)
astronomer/docs: astro/release-notes.md

**Latest CLI Version**: 1.14.1 ([Release notes](cli/release-notes.md))

## May 2, 2023

### Bug fixes

> - Fixed an issue where SSO configurations made through Astronomer support could be overridden by updating the SSO configuration through the Cloud UI.

Author: @vandyliu Is this the right way to describe https://github.com/astronomer/astro/issues/12594?

Reviewer: LGTM

**Latest CLI Version**: 1.14.1 ([Release notes](cli/release-notes.md))

## May 2, 2023

> ### Receive DAG alerts on Slack or PagerDuty

Author: ```suggestion
### Receive Astro alerts on Slack or PagerDuty
```

> DAG alerts are a new way to be notified when your DAGs aren't running as expected. Unlike Airflow callbacks and SLAs, Astro DAG alerts require no changes to DAG code and integrate with Slack and PagerDuty.

Author: ```suggestion
Astro alerts are a new way to be notified when your DAGs aren't running as expected. Unlike Airflow callbacks and SLAs, Astro alerts require no changes to DAG code and integrate with Slack and PagerDuty.
```

> You can set an alert on any DAG to be notified when the DAG fails or when a task takes longer to run than expected. See [Doc link] for configuration steps.

Author: ```suggestion
You can set an alert on any DAG to be notified when the DAG fails or when a task takes longer to run than expected. See [Astro alerts](alerts.md) for configuration steps.
```
